uint8 id
string plant_type
std_msgs/Float64MultiArray weeds
